share/version.py

```diff
@@ -0,0 +1,5 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+
+version = "1.9.0"
```

Comment:
let's make the content of this file updated on deploy time or in CI when we release a new tag. no way we have to remember to change its content on every release :)

Reply:
I agree, however, there are different approaches and caveats to consider:

* Should we have a workflow that is triggered on a `git tag push` event, that bumps this line to the next version? Since we do not have version branches, this wouldn't be that trivial to implement. What do we do if the current version in main is, for example, `1.11.1`, but we want to create a tag for `1.20`?

* Should we bump the version based on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)? E.g we add [bugfix] as prefix to the commit, and this automatically bump the minor version. How do we automatically handle multiple features/commit in a single release?

* We do not provide support for other cloud providers apart from AWS, yet the version file is in the shared package. I would not worry about it right now, as no support for other cloud providers is in the horizon, but I'd keep in mind while choosing the best approach. Related to this, the current tag name pattern does not match what it is written in `version.py` (i.e lambda-vX.X.X instead of vX.X.X)

Reply:
I agree. I moved the discussion on this tracking issue https://github.com/elastic/elastic-serverless-forwarder/issues/540